tests/.eslintrc.js

```diff
@@ -1,5 +1,8 @@
 module.exports = {
   env: {
     'embertest': true
+  },
+  globals: {
+    'expect': true
```

Comment:
also `expect` itself should probably be imported from `chai` instead of using the global, right?